packages/@okta/vuepress-site/docs/guides/client-secret-rotation-key/main/index.md

```diff
@@ -0,0 +1,125 @@
+
+Additionally, depending on what type of credentials that a [client uses to authenticate](/docs/reference/api/oidc/#client-authentication-methods), the use of a JWK public/private key pair may be required. Applications that use private/public key pairs for client authentication have substantially higher security because the private key can only be accessed by the client. But, private/public key pair generation can be laborious and time-consuming, and using the API can lead to errors.
+
+To make client secret rotation more seamless, you can generate an additional client secret for web apps, service apps, and native apps in the Admin Console. You can also generate a JWK public/private key pair (in .JWK format) for your app using the Admin Console.
+
+> **Note:** Using client authentication with a client secret isn’t recommended for native apps because they are public clients. The default authorization type for native apps is **Authorization Code with PKCE**. See [Recommended flow by application type](/docs/concepts/oauth-openid/#recommended-flow-by-application-type) and [Implement authorization by grant type](https://developer.okta.com/docs/guides/implement-grant-type/authcodepkce/main/) for more information on the type of flow to use for your application and how to implement that flow.
+
+### About the Postman collection
+
+It is up to you how you make requests to the APIs to generate client secrets and manage JWKs. In this guide, we provide examples of the required API calls using a Postman collection to demonstrate them in a language/platform neutral way. The collection contains two sub-collections: Secret Management API for client secret generation and management tasks and JSON Web Key (JWK) Management API for JWK management tasks.
+
+## Rotate a client secret
+
+When you are ready to rotate a client secret for an app, follow these steps:
+
+1. Sign in to your Okta organization with your administrator account and go to **Applications** > **Applications**.
+
+2. Select the OpenID Connect app that you want to rotate the client secret for.
+
+3. In the **Client Secrets** section, click **Generate new secret** to create a client secret as a backup to your existing one. A second secret appears with the creation date. The maximum number of secrets that you can generate is two for each app.
+
+    > **Note:** You can try this in Postman using the **Add a client secret: auto-generated** request.
+
+    When you generate a new secret, the original secret remains in **Active** status. Both secrets are stored in parallel, allowing clients to continue using the old secret during secret rotation. Any requests for a secret return the newly generated client secret, but any requests that are sent using the previous secret still work until the status of that client secret is set to **Inactive**.
```

Comment:
This paragraph sounds very important. I wonder if there is a way to set it apart, either as a note or in some other way. I'm a bit concerned that it could get lost here.